testassets/FunctionalTestsWebsite/Greeter.cs

```diff
@@ -39,6 +42,22 @@ public override Task<HelloReply> SayHello(HelloRequest request, ServerCallContex
         return Task.FromResult(new HelloReply { Message = "Hello " + request.Name });
     }
 
+    public override Task<HelloReply> SayHelloWithHttpContextAccessor(HelloRequest request, ServerCallContext context)
+    {
+        var httpContext = _httpContextAccessor.HttpContext;
```

Comment:
I hope this isn't the only way to get the HttpContext

Reply:
https://github.com/grpc/grpc-dotnet/pull/106/files#diff-e2b95dbc2cf449787b017fdd16ca0d35R55 ???

Reply:
Phew

Reply:
The SignalR similarities are uncanny